core/src/main/java/bio/terra/pearl/core/model/i18n/CoreLanguageText.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.pearl.core.model.i18n;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class CoreLanguageText extends BaseEntity {
+    private String messageKey;
```

Comment:
Nit: I find `messageKey` slightly confusing, because it makes me think the text is for translations of an email or some kind of chat message.

Reply:
Good point, I'll rename!